### DISCUSSION TITLE
Youtube playlist

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/OnlineSelfCodingGroup#10] [Ideas]
## Motivation

Inspired @fora22 and @kiJiyeon 

@fora22 님이 전에 음악은 안틀어주냐고 해서 제가 음악은 개인 취향이고 강요해서는 안된다고 해서 알아서 각자 들으면 된다고 했는데
이것 역시 음악을 안듣는게 집중이 잘 되는 저의 취향을 강요한것 같아 생각을 해보게 됐습니다.

또 @kiJiyeon 님이 공부하고 정리한 노션 페이지를 보니 무슨 음악을 들으면서 공부했는지 링크를 달아 놓으셨더라고요

그래서 음악의 취향을 강요하지는 않지만 (물론 음악을 안들을 자유 드림)
모두가 음악의 취향을 강요한다면 강요가 아니라 공유쯤으로 생각해도 되지 않을까 해서

## Youtube playlist

를 만들어 보려고 합니다.

Playlist 추가 링크:
https://www.youtube.com/playlist?list=PLBFc2T4S7JnvZrV92zSlOfqeYHj7-qJfB

여기에 제 음악적 취향을 ~~강요~~ 공유시켜 드릴 노래 3 곡을 넣어 놨습니다.
이 주소에 접속할 수 있는 분 + 구글 계정으로 로그인 한 분은 자유롭게 재생 목록에 영상을 추가할 수 있습니다.
음악 들으실 분은 모임이 시작되기 전에 음악을 담아 두고 1시간 40분 분량의 playlist를 만든 후에  shuffle play를 해보면 어떨까 합니다.

**자 이제 자신의 음악 취향을 다른 사람에게 ~~강요~~ 공유해 주세요.**

**Comment by fora22** [comment]
Discussion  도 새롭고 Jukebox 도 새롭네요! 좋은 아이디어 같습니다!! ㅎㅎ

노래 추가는 어떻게 하나요??

**Reply by jongfeel** [reply]
요기 접속하셔서 구글 로그인을 하시면 자유롭게 추가하실 수 있습니다.

https://www.youtube.com/playlist?list=PLBFc2T4S7JnvZrV92zSlOfqeYHj7-qJfB

**Comment by fora22** [comment]
#16
